spyder/app/tests/test_mainwindow.py

```diff
@@ -3148,5 +3148,24 @@ def test_running_namespace(main_window, qtbot, tmpdir):
     assert nsb.editor.source_model._data['a']['view'] == '10'
     assert nsb.editor.source_model._data['b']['view'] == '10'
 
+@flaky(max_runs=3)
+def test_post_mortem(main_window, qtbot, tmpdir):
```

Comment:
Please mark this test as `slow`. That should avoid the failure we're seeing on the Linux tests.

Reply:
Done. Why would that change the Linux change results?

Reply:
Probably because the main window ends up with focus somehow and that interferes with the fast tests come after it.

That's a common problem on Linux because tests don't run on a real display but on a virtual frame buffer, so things work a bit different there.